packages/network-of-terms-catalog/README.md

```diff
@@ -0,0 +1,39 @@
+# Network of Terms Catalog
+
+This is the catalog of datasets that you can query through the
+[Network of Terms API](https://github.com/netwerk-digitaal-erfgoed/network-of-terms-api).
```

Comment:
Or should we mention all APIs (GraphQL, Recon, and even CLI)?

Reply:
Decided to just link to the root readme, which will point to all of the query capabilities.